packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -67,6 +67,11 @@ contract Setup {
     ///         mutating any nonces. MUST not have constructor logic.
     Deploy internal constant deploy = Deploy(address(uint160(uint256(keccak256(abi.encode("optimism.deploy"))))));
 
+    /// @notice The address of the ForkLive contract. Set into state with `etch` to avoid
+    ///         mutating any nonces. MUST not have constructor logic.
+    ForkLive internal constant forkLive =
+        ForkLive(address(uint160(uint256(keccak256(abi.encode("optimism.forklive"))))));
```

Comment:
nit, we should consider a helper function for the `address(uint160(uint256(keccak256(abi.encode(str)))))` logic, like `function toDeterministicEtchAddress(string)` (or whatever function name)

Reply:
yeah, good call out. I think that'll affect a lot of files though so will do that separately but promptly